mantis-publish/mantis-publish-core/src/main/java/io/mantisrx/publish/proto/MantisServerSubscriptionEnvelope.java

```diff
@@ -52,7 +52,7 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(getSubscriptionList());
+        return Objects.hashCode(getSubscriptionList());
```

Comment:
this changes the `hashCode` implementation slightly. I dont think mantis has any usages where the hashCode is serialized but it'd still be nice to keep it unchanged.